.travis.yml

```diff
@@ -11,8 +11,11 @@ script:
 - bash -c 'time bin/bats --tap test'
 - |
   if [[ "$TRAVIS_OS_NAME" == 'linux' ]]; then
-    docker build --tag bats:latest .
-    docker run -it bats:latest --tap /opt/bats/test
+    for bashver in 3.2 4.0 4.1 4.2 4.3 4.4
+    do
+      docker build --build-arg bashver=${bashver} --tag bats:bash${bashver} .
```

Comment:
I think per #114, the image should be `bats/bats:bash${bashver}`, correct?

Also, I'm partial to adding a dash, i.e. `bats/bats:bash-${bashver}`, but I'm not super concerned about it.

Reply:
I was playing off the previously set versioning in the Travis CI config.  That change is definitely doable.

The added dash is probably a bit more readable, too.